src/Config/CS_SDK.props

```diff
@@ -67,6 +65,7 @@
   <PropertyGroup>
     <BaseIntermediateOutputPath>int\$(TargetFramework)\</BaseIntermediateOutputPath>
     <IntermediateOutputPath>obj\$(TargetFramework)\</IntermediateOutputPath>
+    <TestDependenciesPath>$(OutputPath)\..\..\..\test\test_dependencies</TestDependenciesPath>
```

Comment:
I see some of the packages required for testing are copied to this path but how is it being used?